src/usr/share/gpgmailer/gpgkeyverifier.py

```diff
@@ -217,63 +206,63 @@ def _calculate_recipient_info(self, loop_current_time):
 
         self.valid_recipient_emails = valid_recipient_emails
         self.valid_key_fingerprints = valid_key_fingerprints
+        # Set these as class fields so they can be sent as email body by gpgmailer.
         if all_expiration_warning_messages == '':
             self.expiration_warning_message = None
         else:
             self.expiration_warning_message = all_expiration_warning_messages
 
 
-    # Build an expiration warning message for an individual e-mail and fingerprint pair. There
+    # Build an expiration warning message for an individual fingerprint. There
     #   are essentially two types of messages that can be generated by this method. One is a warning
     #   message that the GPG key is about to expire. The other is a warning message that the GPG
-    #   key has already expired. The returned message will distinguish between sender and recipient
-    #   keys.
+    #   key has already expired. The returned message will distinguish between sender and non-sender
+    #   (potential recipient only) keys.
```

Comment:
I'd probably call this "sender/recipient and recipient only" and get rid of the text in the parenthesis (and the parenthesis themselves. ;)